theano/gpuarray/dnn.py

```diff
@@ -2919,3 +2919,53 @@ def local_gpua_softmax_dnn_grad(op, ctx_name, inputs, outputs):
     out = GpuDnnSoftmaxGrad('accurate', 'instance')(
         gpu_contiguous(ins[0]), gpu_contiguous(ins[1]))
     return [out.dimshuffle(0, 2)]
+
+
+class GpuRoIPool(RoIPoolOp, COp):
```

Comment:
Yes, but I was not familiar with defining new kernels, so I did it this way after having a quick discussion with @nouiz . But I am not sure if this would work as I am facing some errors with Numpy's C- API